helper/consts/agent.go

```diff
@@ -0,0 +1,5 @@
+package consts
+
+// AgentPathCacheClear is the path that the agent will use as its cache-clear
+// endpoint.
+const AgentPathCacheClear = "/agent/v1/cache-clear"
```

Comment:
```suggestion
const AgentPathCacheClear = "/v1/agent/cache-clear"
```

Reply:
It's actually intentional that it's being changed to `/agent/v1` so as to not collide with a potential path on the server when proxying.

Reply:
👍